.github/workflows/im-build-test-deploy.yaml

```diff
@@ -43,7 +43,8 @@ on:
 
 jobs:
   build:
-    if: github.event_name == 'push'
+    env:
+      DEPLOY_ENVIRONMENT: ${{ github.event_name == 'push' || (github.event_name == 'pull_request' && contains(github.event.pull_request.labels.*.name, 'deploy-feature')) }}
```

Comment:
I think we should find a label name without `feature` as I might deploy a bug fix. How about `deploy-im` or simply `deploy` if this is also independent of im 🤔

Reply:
We're deploying to the `instance-manager-feature` namespace, that's why I chose `deploy-feature`. Maybe just `deploy` 🤷

Reply:
@tonsV2 is also in favor of label `deploy` 😄